packages/protocol/test/bridge/Bridge.test.ts

```diff
@@ -1010,11 +1014,675 @@ describe("integration:Bridge", function () {
                 [{ header: blockHeader, proof: encodedProof }]
             )
 
-            // ROGER: this is where we at, we need now to deploy a custom TestHeaderSync that implements
-            // IHeaderSync where we can manually save synced headers.
-            await l2Bridge.processMessage(message, signalProof, {
-                gasLimit: BigNumber.from(2000000),
+            expect(
+                await l2Bridge.processMessage(message, signalProof, {
+                    gasLimit: BigNumber.from(2000000),
+                })
+            ).to.emit(l2Bridge, "MessageStatusChanged")
+        })
+    })
+
+    describe("isMessageSent()", function () {
+        it("should return false, since no message was sent", async function () {
+            const { owner, l1Bridge, srcChainId, enabledDestChainId } =
+                await deployBridgeFixture()
+
+            const m: Message = {
+                id: 1,
+                sender: owner.address,
```

Comment:
we also use this same message struct a lot, perhaps the `deployBridgeFixture` can declare this identical message and return it from there? Woudl remove a lot of lines from each individual test, where we just use same message.